### ISSUE TITLE
Add some quick information about what it's actually doing

### ISSUE THREAD

**Issue by fbernier** [gazay/gon#180]
I just quickly stumbled upon this repository and couldn't find much information about what it's actually doing to make the variables available in the JS. It is something I would want to know before using such a library and I guess having this information in the README would be useful instead of going through the code.

**Comment by gazay** [comment]
Sure! This information is in usage section: https://github.com/gazay/gon#usage

**Comment by fbernier** [comment]
My bad, I went too quickly. Thanks!

**Comment by gazay** [comment]
No problem =)

**Comment by gazay** [comment]
Ah sorry, I went to quickly in issue, so I misunderstood you. Do you mean I need to provide information about internal gon flow, how it stores data from action and then renders it in view? But I thought it is pretty clear – it just stores data for current request inside Gon RequestStore, then when view calls include_gon – it takes it from there and converts to json and add script tag with it.

**Comment by fbernier** [comment]
Yes, this I what I meant. It is indeed quite clear it should be doing what you just described once you read the usage section correctly, but the I believe the short description you just gave could be worthy of being included somewhere.
